specifications/warc-zstd/index.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Zstandard Compression for WARC Files
+status: proposed
+version: 1.0
+numbered: true
+latest: true
+version-of: warc-zstd
+---
+
+# Introduction *(informative)*
+
+This specification defines a Zstandard-based format for compressed WARC files,
+as an alternative to the GZIP-based format defined in [WARC/1.1 Annex D].
+
+In general, Zstandard can produce significantly smaller files than GZIP while
+also achieving faster compression and decompression. Zstandard also offers a
+much wider range of compression levels, ranging from extremely fast compression
+with a modest size reduction to extremely slow compression with a much better
+reduction. For files containing many small records, Zstandard dictionaries can
+be used reduce file size even further, while still permitting random access to
```

Comment:
Typo: "can be used *to* reduce file size"

Reply:
Fixed.